### ISSUE TITLE
Lens correction applied twice?

### ISSUE THREAD

**Issue by sdonath** [Beep6581/RawTherapee#5982]
When I open the attached orf file in RawTherapee, the image looks slightly distorted. When I manually disable Lensfun, the distortion goes away and the image looks very much like the in-camera jpg.
So, is the lens correction applied twice? Maybe once from the information of the raw image and once from Lensfun?

Attached are the raw image, in-camera jpg, RawTherapee jpgs with Lensfun on and off:
https://filebin.net/055f8i6w3377z263

RawTherapee version:
> Version: 5.8
> Branch: releases
> Commit: 9a9e0acbf
> Commit date: 2020-02-04
> Compiler: cc 9.2.0
> Processor: x86_64
> System: Linux
> Bit depth: 64 bits
> Gtkmm: V3.24.2
> Lensfun: V0.3.95.0
> Build type: Release
> Build flags: -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -std=c++11  -Werror=unused-label -Werror=delete-incomplete -Wall -Wuninitialized -Wcast-qual -Wno-deprecated-declarations -Wno-unused-result -fopenmp -Werror=unknown-pragmas -O3 -DNDEBUG -ftree-vectorize
> Link flags: -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now
> OpenMP support: ON
> MMAP support: ON

My operating system:
> Arch Linux
> 5.9.2-arch1-1 #1 SMP PREEMPT Thu, 29 Oct 2020 17:01:28 +0000 x86_64 GNU/Linux

**Comment by ff2000** [comment]
When I align the drawn bottom line to the corner of the window I see distortion in the OOC jpeg.
Doing so with the lensfun corrected image that distortion goes away.

Note that you are using a Panasonic lens on  an olympus camera. I don't know the exact date but up to some point panasonic was shipping the lens correction data with the camera firmware. Which meant that with every new lens every camera firmware had to be updated. At some point they bundled it in the lens. AFAIK Olympus uses the lens-bundeled data when available but has no data for older panasonic lenses in their camera firmware.

I think that the Panasoinc 14-42 II just doesn't ship the lens correction data and therefor the OOC-jpeg is uncorrected.

**Comment by Beep6581** [comment]
> is the lens correction applied twice?

Highly unlikely, as this would show in other cases.

Furthermore, comparing an image I saved with Profiled Lens Correction disabled compared to one with that enabled, the one with it enabled looks slightly better, not worse. I applied neutral, then auto levels, then perspective correction, and finally PLC. Compared using a grid in Inkscape.

![screenshot_20201108_012230](https://user-images.githubusercontent.com/5844619/98454095-f2433580-2160-11eb-880b-387628902d9a.png)
![screenshot_20201108_012237](https://user-images.githubusercontent.com/5844619/98454099-f5d6bc80-2160-11eb-98df-906dce3b2a6c.png)

[PB070439.ORF.pp3.txt](https://github.com/Beep6581/RawTherapee/files/5505692/PB070439.ORF.pp3.txt)
